articles/article-055-turing-completeness.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGIN TITLE -->
+# Is x a more powerful programming language than y?
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+There are hundreds or even thousands of programming languages, but are all of these equally powerful, i.e. can they be used to compute an arbitrary problem? The answer is yes--if the language is Turing complete.
+
+A programming language is called Turing complete if it can be used to simulate a Turing machine. Because the Turing machine is able to compute everything that is computable, a Turing complete language is therefore equally powerful.
```

Comment:
You are not exactly defining a Turing machine here. But I feel like someone else has already wrote on this topic (maybe you can refer to it?). I would change the sentence to:
Since a Turing machine is defined as to be able to compute everything computable, all Turing complete languages are all equally powerful.